lib/engine-jit/Cargo.toml

```diff
@@ -11,16 +11,16 @@ readme = "README.md"
 edition = "2018"
 
 [dependencies]
-wasmer-types = { path = "../types", version = "1.0.2" }
-wasmer-compiler = { path = "../compiler", version = "1.0.2", features = ["translator"] }
-wasmer-vm = { path = "../vm", version = "1.0.2" }
+wasmer-types = { path = "../types", version = "1.0.2", features = ["enable-rkyv"] }
+wasmer-compiler = { path = "../compiler", version = "1.0.2", features = ["translator", "enable-rkyv"] }
+wasmer-vm = { path = "../vm", version = "1.0.2", features = ["enable-rkyv"] }
 wasmer-engine = { path = "../engine", version = "1.0.2" }
 # flexbuffers = { path = "../../../flatbuffers/rust/flexbuffers", version = "0.1.0" }
 region = "2.2"
 serde = { version = "1.0", features = ["derive", "rc"] }
-serde_bytes = { version = "0.11" }
-bincode = "1.3"
 cfg-if = "0.1"
+leb128 = "0.2"
+rkyv = "0.6.1"
```

Comment:
`rkyv` is required here, but it's optinal in `wasmer-compiler`. Why?

Reply:
The engine chooses how to serialize things.
There might be an engine that chooses to serialize with serde, and other that chooses with rkyv. Because of that the serialization mechanism is optional in compiler. But in an engine is mandatory (and unique)